package javaAssignments;

import java.util.Scanner;

public class ExceptionHandling {
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter a");
		int a=sc.nextInt();
		System.out.println("Enter b");
		int b=sc.nextInt();
		int[] arr={1,4,6};
		try{
			double c=a/b;
			System.out.println(arr[5]);
		}
		catch(ArithmeticException e){
			e.printStackTrace();
			System.out.println(e.getMessage());
		}
		catch(ArrayIndexOutOfBoundsException e){
			e.printStackTrace();
			System.out.println(e.getMessage());
		}
	}
}
